app/router/router.tsx

```diff
@@ -153,6 +153,8 @@ export class Path {
   static commitHistoryPath = "/history/commit/";
   static setupPath = "/docs/setup/";
   static settingsPath = "/settings/";
+  static createOrgPath = "/org/create";
```

Comment:
it's gnarly and totally my fault, but you'll need to add `/org/` here so the server knows that's a path on which to serve the app (if you do a full page refresh)

https://github.com/buildbuddy-io/buildbuddy/blob/b10b73c12757eee49b990e619a997857a16726ed/server/libmain/libmain.go#L247

Reply:
Done -- thanks!